src/pages-and-resources/discussions/app-config-form/apps/shared/blackout-dates/BlackoutDatesItem.jsx

```diff
@@ -0,0 +1,153 @@
+import React, { useState } from 'react';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import { Form } from '@edx/paragon';
+import { useFormikContext } from 'formik';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+
+import messages from '../messages';
+import BlackoutDatesInput from './BlackoutDatesInput';
+import { formatBlackoutDates } from '../../../utils';
+import {
+  blackoutDatesStatus as constants,
+  deleteHelperText,
+  badgeVariant,
+} from '../../../../data/constants';
+import CollapsableEditor from '../../../../../../generic/CollapsableEditor';
+import DeletePopup from '../../../../../../generic/DeletePopup';
+import CollapseCardHeading from './CollapseCardHeading';
+
+const BlackoutDatesItem = ({
+  intl,
+  index,
+  blackoutDate,
+  onDelete,
+  hasError,
+  onClose,
+}) => {
+  const blackoutDatesHasError = !blackoutDate.startDate || !blackoutDate.endDate || hasError;
```

Comment:
as discussed we decided to remove this logic and shift our calculation of field error at the upper level so we don't have to check these fields manually --- that feedback has not been addressed.

Reply:
Addressing.